docs/reference/organizations-on-nuget-org.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Organizations on nuget.org | Microsoft Docs
+author: anangaur
+ms.author: anangaur
+manager: unnir
+ms.date: 04/10/2018
+ms.topic: article
+ms.prod: nuget
+ms.technology: null
+description: Organizations on nuget.org helps you to manage packages published by group or in a team, company environment.
+ms.reviewer:
+- karann-msft
+- kraigb
+- camsoper
+---
+
+# Organization on nuget.org
+
+Organizations enables businesses and open-source projects to collaborate across many packages at once using a single nuget.org identity. For a package consumer, an organization account appears same as an existing user account on nuget.org.
```

Comment:
Consider:
Organizations enable businesses and open-source projects to collaborate on packages using a single nuget.org identity. For a package consumer, an organization account appears same as an existing user account on nuget.org.

Reply:
Thanks!